openapi/paths/invoices@{id}@credit-memo-allocations@{creditMemoId}.yaml

```
@@ -16,9 +16,6 @@ get:
   x-sdk-operation-name: getCreditMemoAllocation
   description: >
     Get the allocation from the given credit memo to the given invoice with the credit amount.
```

nitpick: we should start all getter descriptions with "retrieves" ([active verb](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#operations))

I also found it difficult to understand this description. I've tried to make it more clear, not sure if I'm correct 

```suggestion
    Retrieves the allocation from the given credit memo and applies a credit amount to the given invoice.
```

---

Getters can't *apply* anything. Maybe this would make it clearer?
```suggestion
    Retrieves the credit amount allocated to the given invoice from a specific credit memo.
```